ceph-pr-submodules/config/definitions/ceph-pr-commits.yml

```diff
@@ -0,0 +1,54 @@
+- job:
+    name: ceph-pr-submodules
+    project-type: freestyle
+    defaults: global
+    display-name: 'ceph: Pull Request modified submodules'
+    concurrent: true
+    quiet-period: 5
+    block-downstream: false
+    block-upstream: false
+    retry-count: 3
+    properties:
+      - github:
+          url: https://github.com/ceph/ceph/
+    logrotate:
+      daysToKeep: 15
+      numToKeep: 30
+      artifactDaysToKeep: -1
+      artifactNumToKeep: -1
+
+    parameters:
+      - string:
+          name: sha1
+          description: "A pull request ID, like 'origin/pr/72/head'"
+
+    triggers:
+      - github-pull-request:
+          allow-whitelist-orgs-as-admins: true
+          org-list:
+            - ceph
+          trigger-phrase: ''
+          only-trigger-phrase: false
+          github-hooks: true
+          permit-all: true
+          auto-close-on-fail: false
+          status-context: "Unmodifed Submodules"
+          started-status: "checking if PR has modified submodules"
+          success-status: "submodules for project are unmodified"
+          failure-status: "found modified submodules"
```

Comment:
That might be a good idea, but I would prefer that to be an all-or-nothing change across all PR jobs for this project. Otherwise it gets confusing when the messaging is not consistent.